jax/_src/lax/control_flow/loops.py

```diff
@@ -97,15 +98,15 @@ def _promote_weak_typed_inputs(in_vals, in_avals, out_avals):
 
 Carry = TypeVar('Carry')
 X = TypeVar('X')
-Y = TypeVar('Y')
+Any = TypeVar('Any')
 
 @api_boundary
-def scan(f: Callable[[Carry, X], tuple[Carry, Y]],
+def scan(f: Callable[[Carry, X], Tuple[Carry, Any]],
```

Comment:
We should continue to use the built-in (lowercase) `tuple` rather than importing the (upper-case) `Tuple`, because the latter is deprecated in newer Python versions.